test/CleanupService-spec.js

```diff
@@ -10,21 +10,21 @@ const sampleState = {
 	"lists": {
 		"default": [
 			{
-				"regExp": "^.*google\\.com$",
```

Comment:
I'm not totally sure that this is the right way to fix these tests, but I _think_ it is?